client/src/client.rs

```diff
@@ -450,6 +477,16 @@ impl Client {
         self.call("generatetoaddress", &[block_num.into(), address.into()])
     }
 
+    /// Mine up to nblocks blocks immediately (before the RPC call returns)
```

Comment:
```suggestion
    /// Mine up to block_num blocks immediately (before the RPC call returns)
```
Is it worth documenting this call and not the others?

Reply:
I know that others are not commented but I pasted the bitcoin cli help message. For some complex call it can be helpful otherwise you need to open the bitcoin-cli and --help, but for this one it is maybe unnecessary.